robottelo/helpers.py

```diff
@@ -716,3 +716,64 @@ def idgen(val):
     test name
     """
     return '_parameter'
+
+
+class InstallerCommand:
+    """This class constructs, parses, updates and gets formatted installer commands"""
+
+    def __init__(self, *args, command='satellite-installer', allow_dupes=False, **kwargs):
```

Comment:
Parsing methods are hell to debug, can you please add/provide as much documentation as possible.